llvm/cmake/modules/HandleLLVMOptions.cmake

```diff
@@ -149,7 +149,7 @@ else(WIN32)
     endif()
   else(FUCHSIA OR UNIX)
     MESSAGE(SEND_ERROR "Unable to determine platform")
-  endif(FUCHSIA OR UNIX)
+  endif(FUCHSIA OR UNIX OR CHEERP)
```

Comment:
What's the purpose of this?

Reply:
Fix a warning: CMakes says that connected to this line:
if(FUCHSIA OR UNIX OR CHEERP)
saying that the endif-clause do not match the opening clause.
This is minor, and arguably the same fix should also go into line 150.